js/main.js

```diff
@@ -1,72 +1,9 @@
-const getRandomNumber = (minimumValue, maximumValue) => {
-  if(maximumValue <= minimumValue) {
-    throw new Error('Минимальное число, должно быть больше максимального.');
-  }
-  return Math.floor(Math.random() * (maximumValue - minimumValue + 1)) + minimumValue;
-};
-
-
-const getRandomСoordinates = (minimumValue, maximumValue, afterTheComma) => {
-  if(maximumValue <= minimumValue){
-    throw new Error('Минимальное число, должно быть больше максимального.');
-  }
-  const randomСoordinates = Math.random() * (maximumValue - minimumValue + 1) + minimumValue;
-  return Number(randomСoordinates.toFixed(afterTheComma));
-};
-
-
-const getAvatar = () => {
-  const avatarId = getRandomNumber(1, 9);
-  const author = {
-    avatar: `img/avatars/user0${avatarId}.png`};
-  return author;
-};
-
-const TYPES = [
-  'palace',
-  'flat',
-  'house',
-  'bungalow',
-  'hotel',
-];
-
-const CHECKINS = [
-  '12:00',
-  '13:00',
-  '14:00',
-];
-
-const CHECKOUTS = [
-  '12:00',
-  '13:00',
-  '14:00',
-];
-
-const FEATURES = [
-  'wifi',
-  'dishwasher',
-  'parking',
-  'washer',
-  'elevator',
-  'conditioner',
-];
-const DESCRIPTIONS = [
-  'big',
-  'small',
-  'average',
-];
-
-const PHOTOS = [
-  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
-  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
-  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
-];
+import {getRandomNumber, getRandomСoordinates, getRandomOfferElement} from './random';
+import {getAvatar} from './avatar';
+import {TYPES, CHECKINS, CHECKOUTS, FEATURES, DESCRIPTIONS, PHOTOS} from './form-objects';
```

Comment:
```suggestion
import {getRandomNumber, getRandomСoordinates, getRandomOfferElement} from './random.js';
import {getAvatar} from './avatar.js';
import {TYPES, CHECKINS, CHECKOUTS, FEATURES, DESCRIPTIONS, PHOTOS} from './form-objects.js';
```
Без расширений работать не будет 🤔

Reply:
Это нужно исправить везде